twake/frontend/src/app/components/Drive/Drive.scss

```diff
@@ -105,7 +105,7 @@
   flex-direction: column;
   line-height: 30px;
   margin-right: 16px;
-  margin-bottom: 16px;
+  margin-bottom: 4px !important;
```

Comment:
You shouln't need to change the global Drive.scss UI (it can change the UI in the Drive app)
Please cancel changes to this file.

Reply:
Done :+1: